content/tutorials/Data-channels-the-right-way-using-Ably.textile

```diff
@@ -389,6 +389,11 @@ Next, let us update the @recieveMessage@ method to be able to play the notificat
 "See this step in Github":https://github.com/samuelayo/tutorials/commit/044232e714f13975ac32ff0f393114f3160b11fd
 
 h2.
-  default: Step 7 - Conclusion
+  default: Step 8 - Testing our app
```

Comment:
Vague instructions - I think this lesson prepares the users for the subsequent lessons but does not by itself implement any application of WebRTC, correct?
If so, 'Testing instructions' are unnecessary here, but please add them in the lessons necessary. Perhaps, would be good to embed a live demo as well, similar to https://www.ably.io/tutorials/publish-subscribe#live-demo

Reply:
An example of WebRTC live demo: https://deepstreamhub.com/tutorials/protocols/webrtc-screen-sharing/#live-demo

Reply:
Please add a link to the next chapter while mentioning so in one line. Perhaps, 'Next -> XYZ' and link this text to the actual chapter.

Reply:
This lesson implements WebRTC's Datachannels.